pkg/controller/rest/presentproof/operation.go

```diff
@@ -180,9 +180,27 @@ func (c *Operation) AcceptProposePresentation(rw http.ResponseWriter, req *http.
 //    default: genericError
 //        200: presentProofAcceptPresentationResponse
 func (c *Operation) AcceptPresentation(rw http.ResponseWriter, req *http.Request) { // nolint: dupl
+	var buf bytes.Buffer
+
+	if req.Body != nil {
+		// nolint: errcheck
```

Comment:
why skip errcheck?

Reply:
Even if an error occurs the next line checks whether payload `the buf.Bytes()` is a JSON. It will return `InvalidRequestErrorCode` anyway. I skipped it because in my opinion it kind of redundant (I mean in that specific case).